src/main/java/org/gridsuite/modification/server/modifications/OperatingStatusModification.java

```diff
@@ -158,13 +159,17 @@ private void applyEnergiseEquipmentEnd(ReportNode subReportNode, Identifiable<?>
     }
 
     private boolean disconnectAllTerminals(Identifiable<?> equipment) {
-        return ModificationUtils.getInstance().getTerminalsFromIdentifiable(equipment).stream().allMatch(this::disconnectOneTerminal);
+        return ModificationUtils.getInstance().getTerminalsFromIdentifiable(equipment).stream().allMatch(this::disconnectNonFictionalOneTerminal);
     }
 
     private boolean disconnectOneTerminal(Terminal terminal) {
         return !terminal.isConnected() || terminal.disconnect();
     }
 
+    private boolean disconnectNonFictionalOneTerminal(Terminal terminal) {
```

Comment:
Remove `NonFictional` in the name and pass the predicate in argument

Reply:
done